lib/engine/game/base.rb

```diff
@@ -1640,6 +1640,13 @@ def cache_objects
         end
       end
 
+      def clear_cache(type)
+        return unless CACHABLE.any? { |t, _n| t == type }
```

Comment:
i'm a bit wary of this function? what's the intention?

Reply:
Short answer: to clear the cached par prices. 
Longer answer: In 1828, more par prices become available throughout the game and if the cache isn't updated with the new par prices, then share_price_by_id won't be able to get at them. The game crashes on the par action if the screen is refreshed or undo used.

Maybe update_cache is a better name that clear_cache?

Reply:
Essentially, the cache implies all of those objects are immutable. That's not true for all games.

Reply:
ok change to update

Reply:
Change made.